bin.src/phosim_repackager.py

```diff
@@ -22,9 +22,13 @@
 parser.add_argument('--image_type', type=str, default='skyexp',
                     help="image type, passed as 'IMGTYPE': skyexp, dark,\
                     flat, or bias (default: skyexp)")
+parser.add_argument('--focusz', type=float, default=0,
+                    help="The defocal position. For extra-focal it is -1500, for intra-focal 1500,\
```

Comment:
The user can put his/her own `focusz` value. Right? And in this case, maybe you need say the input value should be >=0. Otherwise, your extra- and intra-focal images will have the wrong assignment. In addition, the `1500` here is just a example value. Right?

Reply:
It is the position of the main cam-hexapod (https://confluence.lsstcorp.org/display/SYSENG/ComCam+Header+Information+Topic+Mapping) so it is 0 if in-focus, or +/1 `defocalDistance`, whatever that is. I've clarified the docstring.